docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,539 @@
+# Notification-ex
+
+
+An application can use notifications-ex to inform users about important information.
+The Notification-ex API is designed to create large and expandable notifications.
+
+The main features of the Notification-ex API include:
+
+- Creating a notification-ex item
+
+  You can [create a notification item](#create). There are 12 notification-ex item types.
+  There are five receiver-groups where you want your notification to be displayed on.
+
+- Creating a group item and add a child
+
+  You can [create a group item](#group). You can include multiple items in a group in a specific order.
+  You can create multiple groups and include them in another group.
+
+- Setting notification attributes
+
+  You can [set notification attributes](#set) such as action, style, receiver group, main type, and so on.
+
+- Creating an expanded notification
+
+  You can [create a more extended form](#expanded) of notification.
+
+- Creating a messaging style notification
+
+  You can [create a messaging style](#messaging) notification.
+
+- Creating a notification reporter handle
+
+  You need to [create a reporter handle](#reporter) to post a note.
+  While creating a handle, you can also register callbacks for events and errors.
+
+- Posting a notification
+
+  You can [post single notification and multiple notifications](#post) using `noti_ex_reporter_post()`.
```

Comment:
For line 38 and line 42.
As detailed description is followed by the link, I think we can remove "using `noti_ex_reporter_post()`" and "using `noti_ex_reporter_update()` function" from here.